packages/blade/src/components/Card/Card.tsx

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 import { CardSurface } from './CardSurface';
-import { CardProvider, useVerifyInsideCard } from './CardContext';
+import { CardProvider, useVerifyInsideCard, useVerifyOnlyAllowedComponents } from './CardContext';
```

Comment:
can we rename `useVerifyOnlyAllowedComponents` ➡️ `useVerifyAllowedComponents`?

Reply:
Will do.